### PR TITLE
fix(stop_line): z position of stop point

### PR DESCRIPTION
**Pull request by TomohitoAndo**
Signed-off-by: Tomohito Ando <tomohito.ando@tier4.jp>

## Description
Currently z position of stop line is used as the z position of inserted point, but it causes z position error.
(e.g. when there is a stop line in a slope.)
That resulted in strange pitch due to the high elevation angle. 

z position of stop point is calculated [here](https://github.com/autowarefoundation/autoware.universe/blob/7829c07d9a982054638312a44d4f6c374f8aa04d/planning/behavior_velocity_planner/src/scene_module/stop_line/scene.cpp#L55-L57), so I think [this calculation](https://github.com/autowarefoundation/autoware.universe/blob/7829c07d9a982054638312a44d4f6c374f8aa04d/planning/behavior_velocity_planner/src/scene_module/stop_line/scene.cpp#L67) is not needed.

before this change:
I visualized the output path poses from stop_line module.
image

after this change:
image

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [x] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Comment by TomohitoAndo** [comment]
@takayuki5168 
Could you take a look at this because you modified this last time? :pray:  (and as a code owner @purewater0901 )

File: planning/behavior_velocity_planner/src/scene_module/stop_line/scene.cpp
Comment (takayuki5168) [inline planning/behavior_velocity_planner/src/scene_module/stop_line/scene.cpp:67]: When committing this line, I just followed the original implementation before my PR.
Currently, z position is interpolated from the path. So it makes sense to remove this line.

**Review by takayuki5168 — APPROVED** [review]
LGTM

**Comment by TomohitoAndo** [comment]
@purewater0901 
Could you approve this as a code owner if there are no ploblems?

**Review by purewater0901 — APPROVED** [review]
LGTM

**Pull request merged**
